### PR TITLE
Do not use auto increment value on import workflows

### PR DESCRIPTION
**Pull request by rfehling-mittwald**
Workaround for import workflows on postgre dbs
https://github.com/n8n-io/n8n/issues/2527
https://github.com/typeorm/typeorm/issues/2215

Github issue / Community forum post (link here to close automatically):

(Not tested yet)

### CODE REVIEW

**Comment by Joffcom** [comment]
Hey @rfehling-mittwald,

Thanks for the PR, Is this not already resolved with https://github.com/n8n-io/n8n/pull/5192 or is there more to it?

File: packages/cli/src/databases/entities/ExistingWorkflowEntity.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	Entity,
+	PrimaryColumn,
+} from 'typeorm';
+
+// Clone of WorkflowEntity to allow importing workflows via cli with PostgreSQL
+// https://github.com/n8n-io/n8n/issues/2527
+
+@Entity({ name: "workflow_entity" })
+export class ExistingWorkflowEntity extends WorkflowEntity {
+	@PrimaryColumn({ transformer: idStringifier })
```
Comment (rfehling-mittwald) [inline packages/cli/src/databases/entities/ExistingWorkflowEntity.ts:14]: Removed the @Generated Decorator, thats all

**Comment by rfehling-mittwald** [comment]
I did not see that Merge Request. Well, seems like https://github.com/n8n-io/n8n/pull/5192/files will fix it. This MR can be closed :sweat_smile:

**Comment by Joffcom** [comment]
Hey @rfehling-mittwald,

Sounds good to me, I will close this one for now. Thanks for taking the time to put this together though 🙌🏻

**Pull request closed without merging**